subdomains/Subdomain Alfa/src/main/scala/legacy/SparkOpInstance12.scala

```diff
@@ -0,0 +1,25 @@
+package legacy
+import org.apache.spark.sql.SparkSession
+
+import platform.common_classes.SparkOp
+import org.apache.spark.sql.DataFrame
+import platform.common_classes.Metadata
+import platform.common_classes.RunConfigurations
+
+object SparkOpInstance12 extends SparkOp {
```

Comment:
Should be SubdomainOp - which comes from `import platform.common_classes.SubdomainOp`